test/base/SpecBase.scala

```diff
@@ -49,7 +49,16 @@ trait SpecBase extends PlaySpec with GuiceOneAppPerSuite with TryValues with Moc
     new GuiceApplicationBuilder()
       .overrides(
         bind[DataRequiredAction].to[DataRequiredActionImpl],
-        bind[IdentifierAction].to[FakeIdentifierAction],
+        bind[IdentifierAction].to[FakeIdentifierActionOrganisation],
```

Comment:
Could this have been `bind[IdentifierAction].toInstance(new FakeIdentifierAction(Agent))` which accepts an argument of `AffinityGroup` from applicationBuilder?
Nothing wrong with the current approach just seems a duplication of effort?

Reply:
Done.